.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+##### _You may delete all the italicized instructional headings_
+##### _Please indicate which issue your Pull Request address_
```

Comment:
Should be “addresses”?

Reply:
Found this list of keywords that will make a PR close an issue https://help.github.com/en/articles/closing-issues-using-keywords

Reply:
variations of "closes", "fixes", "resolves" should all work

Reply:
Yep! I just mean the sentence “please indicate which issue your Pull Request _addresses_”. Am I reading it wrong?

Reply:
you're so right